### ISSUE TITLE
Please add and expose Java Heap Size Parameter -Xmx

### ISSUE THREAD

**Issue by Killerjoe** [jokay/docker-ccu-historian#148]
Hello,

first of all, thank you for your work.
I recently was using your docker container on a Raspberry Pi 4 with 4 GB of RAM.
After some time I was unable to establish a ssh connect to the Pi anymore.
I needed to disconnect power of the Pi. After that I was able to establish a new ssh session to the PI.
I found out in the kern.log that there were some Out of Memory situations with the java process of ccu-historian.
Currently I can see that the RAM usage is increasing.

`79923 root     21.5 java -jar /opt/ccu-historian/ccu-historian.jar -config /opt/ccu-historian/config/ccu-historian.config
`

Please add a possiblity to configure the Java Heap Size Memory (-Xmx) exposed in the docker-compose file.

Thank you.
Regards,
Jochen

**Comment by jokay** [comment]
I prepared and tested https://github.com/jokay/docker-ccu-historian/pull/149.

Seems to work so far, did some tests by using `-e CONFIG_JAVA_OPTS="-Xmx100m -Xms100m"`.

```sh
# ps aux | grep "java"
root           9  1.3  1.5 3700768 253016 ?      Sl   22:00   0:11 java -Xmx100m -Xms100m -jar /opt/ccu-historian/ccu-historian.jar -config /opt/ccu-historian/config/ccu-historian.config
```

So it can be included in the next release.

**Comment by jokay** [comment]
Support for `CONFIG_JAVA_OPTS` is merged and will be available in the next release, see [here](https://github.com/jokay/docker-ccu-historian/blob/main/CHANGELOG.md).

**Comment by Killerjoe** [comment]
Thank you very much for fast adding this feature!
I hope you will build the next docker container soon, so that I can try and use it.
Regards,
Jochen

**Comment by jokay** [comment]
My releases are based on the releases of CCU-Historian so I can't give you any ETA :wink:

**Comment by PaulMerk** [comment]
@Jokay
Just my 2 cents:
Take your time. 
The conversion of the current database V2.8.3 to the new format of V3 may become demanding.
Paul

**Comment by jokay** [comment]
This will be a manual step, see [here](https://github.com/mdzio/ccu-historian/issues/310#issuecomment-988654037) :wink:

**Comment by jokay** [comment]
Feature is included in todays [2.9.0](https://github.com/jokay/docker-ccu-historian/releases/tag/2.9.0) release.
